components/CircleButton.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Button } from 'react-native-elements';
+import { Text, View } from 'react-native';
+
+export default class CircleButton extends Component {
+  render() {
+    return (
+          <Button
+              title={this.props.title}
```

Comment:
Can you include the defaultProps library so that it's easy to see what props each component needs to take in.